packages/dd-trace/src/llmobs/constants.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+module.exports = {
+  SPAN_KINDS: ['llm', 'agent', 'workflow', 'task', 'tool', 'embedding', 'retrieval'],
+  SPAN_KIND: '_ml_obs.meta.span.kind',
+  SESSION_ID: '_ml_obs.session_id',
+  METADATA: '_ml_obs.meta.metadata',
+  METRICS: '_ml_obs.metrics',
+  ML_APP: '_ml_obs.meta.ml_app',
+  PROPAGATED_PARENT_ID_KEY: '_dd.p.llmobs_parent_id',
+  PARENT_ID_KEY: '_ml_obs.llmobs_parent_id',
+  TAGS: '_ml_obs.tags',
```

Comment:
Nitpick (not a blocker) but I think splitting the constants in multiple files by subtypes (for example `tags.js`) would make them clearer and discovery easier.

Reply:
Ah yes this would be good! I'll do this